release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -238,6 +238,68 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+==== Beta APIs removed from Kubernetes v1.22
+
+The upgrade to Kubernetes v1.22 removed the following deprecated
+`v1beta1` APIs. Migrate manifests and API clients to use the `v1` API
+version. For more information on migrating deprecated APIs, see the
+https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22[Kubernetes
+documentation].
```

Comment:
Also, we might want to check on if these links to Kubernetes are okay. Probably fine, but still good to check. We might also end up with some OCP docs related to this that we can link to.

Reply:
> I thought there was at least one OpenShift-specific v1beta1 API being removed (for the descheduler).

@bergerhoffer Do you know if the descheduler API v1beta1 be considered a [Tier 2 supported API](https://docs.openshift.com/container-platform/4.8/rest_api/understanding-api-support-tiers.html#mapping-support-tiers-to-api-groups_understanding-api-tiers)? If so, I could rename the table to something like "`Tier 2` APIs removed in OpenShift 4.9" and update the intro paragraph to say the APIs are replaced by `Tier 1` counterparts.

Reply:
> Also, we might want to check on if these links to Kubernetes are okay. 
This section of the k8s docs is [referenced in our docs](https://docs.openshift.com/container-platform/4.8/operators/operator_sdk/osdk-working-bundle-images.html#osdk-control-compat_osdk-working-bundle-images) already, so I figured it would be okay. But I will be sure to follow up to make sure.

Reply:
I checked with eng and he thought it was probably tier 3 or 4. So I say to just keep focusing on kubernetes here until we know if there are any more other than the descheduler. We can just do a one off notification for that one if needed.